packages/devextreme/js/__internal/core/state_manager/controller_registry.ts

```diff
@@ -0,0 +1,66 @@
+import { EventEmitter } from './event_emitter';
+import type { IController, IControllerRegistry, ILogger } from './interfaces';
+
+export class ControllerRegistry implements IControllerRegistry {
+  private controllers: Record<string, IController> = {};
+
+  private readonly logger: ILogger;
+
+  private readonly controllerRegisteredEmitter: EventEmitter<
+    (controller: IController, id: string) => void>;
+
+  constructor(logger: ILogger) {
+    this.logger = logger;
+    this.controllerRegisteredEmitter = new EventEmitter<(controller: IController,
+    id: string) => void
+      >(
+      'controllerRegistered',
+      logger,
+      );
```

Comment:
It's really hard to read.

```suggestion
    this.controllerRegisteredEmitter = new EventEmitter<(controller: IController, id: string) => void>(
      'controllerRegistered',
      logger,
    );
```

or 

```ts
type ControllerEventEmitter = (controller: IController, id: string) => void;
```

```suggestion
    this.controllerRegisteredEmitter = new EventEmitter<ControllerEventEmitter>(
      'controllerRegistered',
      logger,
    );
```

Reply:
changed